.gitignore

```diff
@@ -1,2 +1,17 @@
 target/
 .idea
+
+step/portfolio/src/main/java/com/google/sps/data/
```

Comment:
Instead of adding specific file names I suggest adding patterns.  For example:
*.jar
*.tar.gz

Also, you will need to explicitly remove the jar files that you have already added with "git rm". Note that it will also delete the file from you working directory. So, if you want to keep it around in your local directory then make a copy of it first.